Removed null constraints on NodeInfo fields that are returned null from Amazon OpenSearch Service (#1132)
opensearch-project/opensearch-java: CHANGELOG.md

### Removed

### Fixed

> - Fixed handling of NodeInfo fields that are not returned by Amazon OpenSearch Service ([#1131](https://github.com/opensearch-project/opensearch-java/pull/1131))

Reviewer: Sorry I missed catching this before, it should reference this PR rather than the previous one
```suggestion
- Fixed handling of NodeInfo fields that are not returned by Amazon OpenSearch Service ([#1132](https://github.com/opensearch-project/opensearch-java/pull/1132))
```

Author: Oops, good catch! I made the changes.

### Fixed
- Fixed error when deserializing a normalizer without 'type' ([#1111](https://github.com/opensearch-project/opensearch-java/pull/1111))
- Fixed deserialization of SearchRequest when `_source` is an array ([#1117](https://github.com/opensearch-project/opensearch-java/pull/1117))

> - Removed null constraints on NodeInfo fields that are returned null from Amazon OpenSearch Service ([#1131](https://github.com/opensearch-project/opensearch-java/pull/1131))

Reviewer: I think we can simplify this and follow the verbiage of the particular section ("Fixed")
```suggestion
- Fixed handling of NodeInfo fields that are not returned by Amazon OpenSearch Service ([#1131](https://github.com/opensearch-project/opensearch-java/pull/1131))
```
